docs/install.md

```diff
@@ -25,6 +25,33 @@ ensure that your Firedrake environment is activated:
 
 Furthermore, the `gadopt` Python module should be available for you to use.
 
+## Using the G-ADOPT library on Gadi
+
+The G-ADOPT team maintains an up-to-date Firedrake and G-ADOPT installation 
+on NCI's Gadi HPC system. To use G-ADOPT on Gadi, create an NCI account, then follow [this link](https://my.nci.org.au/mancini/project/fp50/join) to request to join the fp50 project. Once your request is approved, you can
+access G-ADOPT on Gadi by running:
+
+    module use /g/data/fp50/modules
+    module load firedrake
+
+To use G-ADOPT in you batch scripts, you'll need to add the following to your
```

Comment:
Can we add an example batch script somewhere? Not sure if this website is the best place or just a link to somewhere in the repo or wiki?

Reply:
At the least I think it would be a good idea to have a code snippet with the line to execute the python script maybe. I guess something like

```
mpiexec -np ${PBS_NCPUS} python3 example_script.py

```

Reply:
Also:
```suggestion
To use G-ADOPT in your batch scripts, you'll need to add the following to your
```